flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -331,6 +331,22 @@
                                     + "JobManager could be faster, since no reverse DNS lookup is performed. "
                                     + "However, local input split assignment (such as for HDFS files) may be impacted.");
 
+    @Documentation.Section(Documentation.Sections.EXPERT_JOB_MANAGER)
+    public static final ConfigOption<Integer> JOB_MANAGER_FUTURE_THREADS =
+            key("jobmanager.future-executor.numThreads")
+                    .intType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "The number of jobmanager future-executor threads for shared services.");
+
+    @Documentation.Section(Documentation.Sections.EXPERT_JOB_MANAGER)
+    public static final ConfigOption<Integer> JOB_MANAGER_IO_THREADS =
+            key("jobmanager.io-executor.numThreads")
+                    .intType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "The number of jobmanager io-executor threads for shared services.");
+
```

Comment:
I think that we also need to rebuild the documentation via `mvn package -Dgenerate-config-docs -pl flink-docs`. Otherwise the configuration options won't appear in the documentation and the documentation completeness test will fail.

Reply:
Was not aware, rebuilding!

Reply:
Rebuilt!